### DISCUSSION TITLE
How to get rid of hydration error on my next js project?

### DISCUSSION THREAD

**Discussion by Saidur07** [vercel/next.js#40088] [Help] [answered]
How to get rid of hydration error on my next js project?

**Comment by mahfuz47** [comment, marked as answer]
you can follow this documentation:
https://nextjs.org/docs/messages/react-hydration-error

**Comment by icyJoseph** [comment]
Hi,

What type of hydration error are you seeing? Mismatched HTML? Or is it perhaps code that's causing errors while rendering on the client? 

Got a repo, or link to an application, to take a look at?
